pages/adminDashboard.js

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { allProfilesInfo } from '../graphql/api';
+
+const getProfiles = (data) => {
+  return data ? data.allProfilesInfo.data : [];
+}
+
+const getInterviews = (applications) => {
+    let interviews = [];
+    applications.forEach(application => {
+        const interview = {
+            _id: application.interviews.data[application.interviews.data.length-1]._id,
```

Comment:
given we're reusing `application.interviews.data.length-1` a couple times, perhaps we should store it in a `const`. it'll make it easier to update the value in the future and help with readability.